src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -804,11 +806,19 @@ public function Breadcrumbs($unlinked = false)
             $items = new ArrayList();
         }
 
+        $params = $this->getRequest()->getVars();
+        if (isset($params['url'])) {
+            unset($params['url']);
+        }
```

Comment:
What are we doing this for?

Reply:
I took this logic from ModelAdmin Breadcrumbs method.